acpi-video-intel-backlight.conf

```diff
@@ -0,0 +1,6 @@
+notify 100 {
+        match "system" "ACPI";
+        match "subsystem" "Video";
+        match "type" "brightness";
+        action "/usr/local/bin/intel_backlight $notify";
```

Comment:
From the ports perspective, it would be nice to be able to replace /usr/local with LOCALBASE during building.

Reply:
I guess I could do that.  It makes the port slightly messier though; I was planning on adding this file to PORTEXAMPLES so that it would be installed along with the two shell scripts.

Reply:
Oh that's probably a fine approach as well.

Reply:
On further consideration, I'd prefer to keep `/usr/local` hard-coded here and have the port replace it by PREFIX.  Can you pull this now?

Reply:
I cannot merge any changes here. I'm just a watcher ;)